epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityTracker.java

```diff
@@ -82,6 +83,8 @@ private static void setTracker(
 
   private boolean onChangedEnabled = true;
 
+  private int partialImpressionThresholdPercentage = 0;
```

Comment:
should it default to something, or default to null in which case `handlePartialImpressionVisible` is not called?

Reply:
Good point. Let's default to `null` and not call the method as you described.

Reply:
I guess I see that it shortcircuits to doing nothing when set to 0, in which case the listeners are never called. i think this is a little confusing since a threshold of `0` seems like it would always trigger it to be called. maybe use -1 or null instead?

Reply:
The other idea is to just not call `handlePartialImpressionVisible` in case of `0`

Reply:
If using -1 to disable the events, would you prefer a separate method to disable the events then in regards to public API? Or just annotate `setPartialImpressionThresholdPercentage`'s parameter with @IntRange(from = -1, to = 100) (seems a bit odd)

Reply:
A value of `0` remains a special case, as it would have to fire if at least one pixel of the element is visible on the screen (would act the same as `VISIBLE`).

Reply:
How about b6d43a5b10b06c43115a6942fd84d2c9962fffb1

Reply:
How about `@IntRange(from = 1, to = 99)`, possible values would be `null` or any `int` from 1 to 99
Btw, if we have `100` we could use the full impression event so `100` is not really needed.

Reply:
I would leave this decision to you as the maintainers of the library as it is more about the public API and what is expected here.

for sure the extremes will basically duplicate existing events:
- `0` acts like `VISIBLE`
- `100` acts like `FULL_IMPRESSION_VISIBLE` (with an added `_INVISIBLE` state)

I think having the full range from 0 to 100 grew on me. I think of it this way: If other visibility events were to drop out and be removed, this one should still be fully functional and in that case 0 and 100 are valid use cases.

Reply:
I don't have a strong opinion on it and since you handle `0` and `100` having `@IntRange(from = 0 to = 100)` is fine.